bindings/generator/api/common.py

```diff
@@ -268,6 +272,20 @@ class InvitationToken(StrBasedType):
     )
 
 
+class EmailValidationToken(StrBasedType):
+    custom_from_rs_string = "|s: String| -> Result<libparsec::EmailValidationToken, _> { libparsec::EmailValidationToken::from_hex(s.as_str()).map_err(|e| e.to_string()) }"
+    custom_to_rs_string = (
+        "|x: libparsec::EmailValidationToken| -> Result<String, &'static str> { Ok(x.hex()) }"
+    )
+
+
+class AccountAuthMethodID(StrBasedType):
```

Comment:
You can remove this since we don't need expose the ID to the bindings

The only part where it could be useful to expose this ID would be in `vault_item_recovery_list`, but currently we just don't return it ;-)